CHANGELOG.md

```diff
@@ -1,5 +1,77 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.4
+
+<!-- generated comparing v5.2.3..master -->
+
+_Dec 14, 2021_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Add `not` operator to `theme.breakpoints` (#29311) @Philipp000
```

Comment:
This could have been considered a new feature => 5.3.0

Reply:
Speaking of new minors, we don't have v5.1 in https://mui.com/versions/. Should we bother?

Reply:
> Should we bother about minor versions? 

I noticed it too, turn out that was the version deployed when the docs were broken (some issue with the translations and infinite refresh). I didn't push further to create docs for that version until someone complains that they need it.